librz/analysis/similarity.c

```diff
@@ -420,7 +422,26 @@ static void *analysis_match_basic_blocks(SharedContext *shared) {
  */
 RZ_API RZ_OWN RzAnalysisMatchResult *rz_analysis_match_basic_blocks(RZ_NONNULL RzAnalysisFunction *fcn_a, RZ_NONNULL RzAnalysisFunction *fcn_b, RZ_NONNULL RzAnalysisMatchOpt *opt) {
 	rz_return_val_if_fail(opt && opt->analysis_a && opt->analysis_b && fcn_a && fcn_b, NULL);
-	return analysis_match_result_new(opt, fcn_a->bbs, fcn_b->bbs, (RzThreadFunction)analysis_match_basic_blocks, (AllocateBuffer)basic_block_data_new);
+
+	RzList *bbs_list_a = rz_list_new();
+	RzList *bbs_list_b = rz_list_new();
+	if (!bbs_list_a || !bbs_list_b) {
+		// If memory allocation failed, print an error message and return NULL
+		eprintf("Failed to allocate memory for bbs_list_a or bbs_list_b.\n");
```

Comment:
RZ_LOG_ERROR

Reply:
use `eprintf` only for debugging.

Reply:
Sure I will resolve this

Reply:
keep this function using RzList. i will submit a PR for allowing Queue with RzPVector, which will help the conversion.